src/Types/Type.php

```diff
@@ -69,11 +71,189 @@ abstract class Type extends BaseType implements NullableType
     public static function get(string $name): NullableType
     {
         if (!isset(static::$types[$name])) {
-            throw new RuntimeException('Unknown type:' . $name);
+            static::$types[$name] = static::create($name);
         }
         return static::$types[$name];
     }
 
+    protected static function create(string $name): NullableType
+    {
+        return match ($name) {
+            '_PaginatorInfo' => static::paginatorInfo(),
+            '_SQLOperator' => static::sqlOperator(),
+            '_CountryCode' => static::countryCodeEnum(),
+            '_LanguageCode' => static::languageEnum(),
+            '_CurrencyCode' => static::currencyEnum(),
+            '_LocaleCode' => static::localeEnum(),
+            '_SortOrder' => static::sortOrderEnum(),
+            '_FileExport' => static::fileExport(),
+            '_ExportFile' => static::sheetFileEnum(),
+            '_ImportSummary' => static::importSummary(),
+            '_ImportError' => static::importError(),
+            '_Result' => static::result(),
+            '_DateTime' => static::dateTime(),
+            '_Date' => static::date(),
+            '_Time' => static::time(),
+            '_TimezoneOffset' => static::timezoneOffset(),
+            'Mixed' => static::mixed(),
+            '_UpdateManyResult' => static::updateManyResult(),
+            '_RelationUpdateMode' => static::relationUpdateModeEnum(),
+            '_Upload' => static::upload(),
+            '_File' => static::file(),
+            '_Job_Column' => static::column('Job_'),
+            '_Job_Status' => static::jobStatus(),
+            '_History_Column' => static::column('History_'),
+            '_History_ChangeType' => static::historyChangeType(),
+            '_History' => static::history(),
+            '_Permission' => static::permissionEnum(),
+            '_Entity' => static::entity(),
+            '_WhereMode' => static::whereMode(),
+            '_Model' => static::modelEnum(),
+            default => static::createDynamic($name),
+        };
+    }
+
+    protected static function createDynamic(string $name): NullableType
+    {
+        // WARNING: order of matching is partially important below!
+        // p.ex.: _<something>EdgeColumn has to be matched before _<something>Column!
+
+        if (str_contains($name, '__')) { // if it does not, skip all these checks
+
+            // _<model>__<field>Enum || _<model>__<relation>__<field>Enum
+            if (DynamicEnum::nameMatches($name)) {
```

Comment:
nester :stuck_out_tongue_closed_eyes:

Reply:
habs mir überlegt den ganzen __ zweig in ne eigene methode rauszuziehen... aber im alten code hat er nach dem __ zeug, die regulären typen weitergematched... und ich war mir net sicher ob ich das gefahrlos entfernen kann